CRM/Contact/Page/DedupeMerge.php

```diff
@@ -140,7 +136,7 @@ public static function getRunner() {
    * @return int
    */
   public static function callBatchMerge(CRM_Queue_TaskContext $ctx, $rgid, $gid, $mode = 'safe', $batchLimit, $isSelected, $criteria) {
-    CRM_Dedupe_Merger::batchMerge($rgid, $gid, $mode, $batchLimit, $isSelected, $criteria);
+    CRM_Dedupe_Merger::batchMerge($rgid, $gid, $mode, $batchLimit, $isSelected, $criteria, TRUE, FALSE);
```

Comment:
callBatchMerge is only called from CRM_Contact_Page_DedupeMerge::getRunner. $batchLimit is hard-coded to self::BATCHLIMIT (2)  which results in $reloadCacheIfEmpty being set to FALSE in CRM_Dedupe_Merger::batchMerge

```

    $redirectForPerformance = ($batchLimit > 1) ? TRUE : FALSE;

    if (!isset($reloadCacheIfEmpty)) {
      $reloadCacheIfEmpty = (!$redirectForPerformance && $isSelected == 2);
    }
```

This is totally bizarre special sauce so I added an override param to start grandfathering it out.